src/api.ts

```diff
@@ -329,6 +328,7 @@ class RippleAPI extends EventEmitter {
   dropsToXrp = dropsToXrp
   iso8601ToRippleTime = iso8601ToRippleTime
   txFlags = txFlags
+  schemaValidator = schemaValidator
```

Comment:
Since this doesn't rely on any options or connect to a rippled server, I would suggest making this `static` so that it can be accessed as `RippleAPI.schemaValidator`.
```
static schemaValidator = schemaValidator
```